paf-mvp-frontend/src/lib/paf-lib.ts

```diff
@@ -757,3 +758,7 @@ export const getAuditLogByDivId = (divId: DivId): AuditLog | undefined => {
   }
   return getAuditLogByTransaction(prebidTransactionId);
 };
+
+// Set up the queue of asynchronous commands
+export const queue: CommandQueue = window.PAF.queue || [];
```

Comment:
I think it would make more sense to write it this way:
```typescript
export const queue: CommandQueue = window.PAF.queue || new CommandProcessor();
processCommands(queue);
```
Otherwise, what's the point to have the `processCommands` method accept `CommandProcessor` as param?
The way it is written today, it will always fall into the `if (queue instanceof CommandProcessor) {` case.
I think it's error prone to both **export** `queue` constant, and **re-affect** `window.PAF.queue` after the call to `processCommands`. Since I believe it is useless anyway, I suggest to drop it.

Reply:
After double thinking, I think you need to update the global variable and make sure it's a processor:

```typescript
if (window.PAF.queue) {
	processCommands(window.PAF.queue);
}
// Reset it once it's been processed
window.PAF.queue = new CommandProcessor();
```